go.mod

```diff
@@ -55,4 +55,7 @@ require (
 	google.golang.org/grpc v1.37.0 // indirect
 )
 
-replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+replace (
+	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+ 	github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
```

Comment:
I'm wondering if latest tag of keyring (https://github.com/99designs/keyring/tree/v1.1.6) already has a fix for this, can you please check?

Reply:
@ilgooz, the problem persists on the v1.1.6:

![Screen Shot 2021-06-21 at 17 34 50](https://user-images.githubusercontent.com/2406457/122824649-557c7500-d2b7-11eb-8b7d-728c218215e0.png)